External/Plugins/ResultsPanel/Settings.cs

```diff
@@ -25,6 +26,14 @@ public GroupingMethod DefaultGrouping
             get { return defaultGroup; }
             set { defaultGroup = value; }
         }
+
+        [DisplayName("Highlight Only Active Panel Entries")]
+        [LocalizedDescription("ResultsPanel.Description.HighlightOnlyActivePanelEntries"), DefaultValue(true)]
```

Comment:
Shouldn't the default be false as the backing field? If you really intended it to be true, I prefer it to be false.